<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="GURL5CG207721Z" failures="1" tests="4" name="Personal Loan" time="0.16" errors="0" timestamp="2024-07-24T13:17:01 IST">
  <testcase classname="test.Day1" name="Demo" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.Demo(Day1.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.Day2" name="ploan" time="0.007"/>
  <testcase classname="test.Day4" name="MobileLoginHomeLoan" time="0.004"/>
  <testcase classname="test.Day4" name="WebLoginHomeLoan" time="0.007"/>
  <testcase classname="test.Day1" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
  <testcase classname="test.Day4" name="LoginAPIHomeLoan" time="0.0">
    <ignored/>
  </testcase> <!-- LoginAPIHomeLoan -->
</testsuite> <!-- Personal Loan -->
